src/assets/js/dynamic-addon-cards.js

```diff
@@ -25,10 +25,11 @@
     getFirefoxButton.querySelector('.GetFirefoxButton-callout').remove();
 
     const button = getFirefoxButton.querySelector('.GetFirefoxButton-button');
+    const isStaticTheme = addon.type === 'statictheme';
```

Comment:
Sorry, I probably commented on/asked about this before, but shouldn't we be using constants for important strings like this?

Reply:
we should eventually but for now, I think it's used twice, which is almost less chars than `ADDON_TYPE_STATICTHEME`